src/pivotal-ui/components/tables/tables.scss

```diff
@@ -73,12 +73,129 @@ table, .table {
 
 /*doc
 ---
-title: Data
+title: Rounded Table
+name: 1_table_round
+parent: table
+---
+
+This is a table used to display many rows of data, with headers and
+footers. Use it for tables of information or associations such as
+user lists with permissions and action menus
+
+```html_example
+<table class="table table-rounded table-header-gray-10">
+  <thead>
+    <tr>
+      <th>Wombat Type</th>
+      <th>Surprisingness</th>
+      <th>Cuteness</th>
+    </tr>
+  </thead>
+  <tbody class="bg-neutral-11">
+    <tr>
+      <td>Baby Wombat</td>
+      <td>100</td>
+      <td>Infinity</td>
+    </tr>
+    <tr>
+      <td>World's Largest</td>
+      <td>1000</td>
+      <td>99.5</td>
+    </tr>
+    <tr>
+      <td>In native habitat</td>
+      <td>12</td>
+      <td>315</td>
+    </tr>
+  </tbody>
+  <tfoot>
+    <tr>
+      <th>Average</th>
+      <td>506</td>
+      <td>Infinity</td>
+    </tr>
+  </tfoot>
+</table>
+```
+
+*/
+
+.table-rounded {
```

Comment:
I wouldn't call this `rounded`, designers love to change the design. Maybe try something abstract that relates to the relationship between this table and another table. e.g. is it more or less important than a basic table? Simple? Basic? Highlight? Alternate? All words that define the relationship without referring to the way it looks.